src/components/Accordion/Accordion.tsx

```diff
@@ -10,10 +10,35 @@ import { Node } from "@react-types/shared";
 import { FOCUS_STYLE_INSET } from "@utilities/focusStyle";
 import { merge } from "@utilities/merge";
 import { AnimatePresence, motion } from "framer-motion";
-import React, { Children, FC, isValidElement, PropsWithChildren, ReactElement, ReactNode, useRef } from "react";
+import React, {
+    Children,
+    FC,
+    isValidElement,
+    KeyboardEvent,
+    PropsWithChildren,
+    ReactElement,
+    ReactNode,
+    useRef,
+} from "react";
 
 export type AccordionItemProps = PropsWithChildren<{ header: FieldsetHeaderProps }>;
 
+const ACCORDION_ID = "accordion";
+const ACCORDION_ITEM_ID = "accordion-item";
+
+const ACCORDION_CONTROL_KEYS = [
+    "ArrowDown",
+    "ArrowUp",
+    "ArrowRight",
+    "ArrowLeft",
+    "Home",
+    "End",
+    "PageDown",
+    "PageUp",
+    "a",
```

Comment:
For what purpose?

Reply:
useAccordion uses a keyboard listener which prevents default of these keys, to keep this consistent with the hook you have to manually call it. The main reason is because pressing some of these keys will activate the window scroller, like the up and down arrow key which is not the expected behaviour from the useAccordion hook

Reply:
Ok, but why the letter `a`? I can understand all the others ones

Reply:
Ah that was used for their selectAll function, but I will remove that it shouldn't change anything